source/slang/slang-reflection-api.cpp

```diff
@@ -2298,6 +2363,20 @@ SLANG_API SlangInt spReflectionTypeLayout_getFieldBindingRangeOffset(SlangReflec
     return 0;
 }
 
+SLANG_API SlangInt spReflectionTypeLayout_getExplicitCounterBindingRangeOffset(SlangReflectionTypeLayout* inTypeLayout)
+{
+    auto typeLayout = convert(inTypeLayout);
+    if(!typeLayout) return 0;
+
+    if(const auto structuredBufferTypeLayout = as<StructuredBufferTypeLayout>(typeLayout))
+    {
+        getExtendedTypeLayout(structuredBufferTypeLayout);
+        return structuredBufferTypeLayout->counterVarLayout->bindingRangeOffset;
```

Comment:
What if `counterVarLayout` is nullptr? Is `counterVarLayout->bindingRangeOffset` always 1 if it exists?